vendor/k8s.io/kubernetes/pkg/volume/glusterfs/glusterfs.go

```diff
@@ -476,9 +484,9 @@ func (r *glusterfsVolumeProvisioner) Provision() (*api.PersistentVolume, error)
 		return nil, err
 	}
 	r.provisioningConfig = *cfg
-
 	glog.V(4).Infof("glusterfs: creating volume with configuration %+v", r.provisioningConfig)
-	glusterfs, sizeGB, err := r.CreateVolume()
+	reqGid = gidMin + gidRandomizer.Int63n(gidMax)
```

Comment:
@eparis  let me try to summarize  :) 
1) with the current version of this patch,  admin dont have control on `gidMin` and `gidMax` (  2000 and 600000) values . As mentioned here,  https://github.com/openshift/origin/pull/11893#discussion_r88827420 , I will make this `GID range` configurable in SC so that admin get control over it.
2)  It seems that , the reservation for SGIDs are 
`openshift.io/sa.scc.supplemental-groups=1000000000/10000` , looks like making these value as default range for GID would be better,  so that if admin didnt mention the range of GID  in SC parameter list, we could use one in the default range.  Please share your thought :) and please feel free to point out if I am missing something here.